business/checkers/common/multi_match_selector_checker_test.go

```diff
@@ -111,15 +127,15 @@ func TestSidecarsDifferentNamespaces(t *testing.T) {
 		}, data.CreateSidecar("sidecar1", "bookinfo")),
 		data.AddSelectorToSidecar(map[string]string{
 			"app":     "details",
-			"version": "v1",
+			"version": "v2",
 		}, data.CreateSidecar("sidecar2", "bookinfo2")),
 		data.AddSelectorToSidecar(map[string]string{
 			"app":     "details",
-			"version": "v1",
+			"version": "v3",
 		}, data.CreateSidecar("sidecar3", "bookinfo3")),
 		data.AddSelectorToSidecar(map[string]string{
 			"app":     "details",
-			"version": "v1",
+			"version": "v4",
```

Comment:
Nitpick: if I understand correctly, `v4` does not exist in the list returned by `workloadList()`...
It is a very small nit. It does not affect the tests. So, take this as a simple comment.

So, no action needed.

If you do want to fix, perhaps do it on another PR as a side change (even if it is an unrelated change). Let's just merge this as is.

Reply:
actually I did it on purpose, to make sure that tests will fail if future changes cause bug